ui/shared/utils/constants.js

```diff
@@ -1188,7 +1188,7 @@ export const PREDICTOR_FIELDS = [
   { field: 'mut_taster', group: MISSENSE_IN_SILICO_GROUP, indicatorMap: MUTTASTER_MAP },
   { field: 'fathmm', group: MISSENSE_IN_SILICO_GROUP, indicatorMap: INDICATOR_MAP },
   { field: 'apogee', warningThreshold: 0.5, dangerThreshold: 0.5 },
-  { field: 'gnomad_noncoding', displayOnly: true, warningThreshold: 0, dangerThreshold: 1.5 },
+  { field: 'gnomad_noncoding', fieldTitle: 'gnomAD Constraint', displayOnly: true, warningThreshold: 0, dangerThreshold: 1.5 },
```

Comment:
The ticket requests a new text, 'gnomAD genomic constraint'. Is it okay to remove the 'genomic'?

Reply:
yes because it looked terrible